06-developer-software-guide.Rmd

```diff
@@ -11,46 +11,25 @@ The FIMS repository already has a file to set up all of the needed software, so
 
 To start a codespace on the FIMS repository, follow the [GitHub documentation for creating a Codespace](https://docs.github.com/en/codespaces/developing-in-a-codespace/creating-a-codespace-for-a-repository?tool=webui#creating-a-codespace-for-a-repository).
 
-## Software to Install
+## Software to install
 
 ### git
 
-You will need git installed locally, and you may prefer to use an
-additional git [GUI client](https://git-scm.com/downloads/guis) such as
-GitKraken or GitHub Desktop. If your preferred git client is the RStudio
-IDE, you can configure Git and RStudio integration following these
-[instructions](https://happygitwithr.com/rstudio-git-GitHub.html).
+To contribute to the code, you will need git installed locally, and you may prefer to use an additional git [GUI client](https://git-scm.com/downloads/guis) such as GitKraken or GitHub Desktop. See the section on [RStudio](#rstudio) for setting up RStudio to work with git.
 
-To install git, please follow the instructions on [this
-page](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) for
-your operating system. You can find the downloads for your operating
-system on the left-hand navigation bar on that page.
+To install git, please follow the operating-system specific instructions available in the [git documentation](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
```

Comment:
can we change to `please contact your IT department or follow...`, as I think it may require admin permissions.

Reply:
Done